server/resources.py

```diff
@@ -22,5 +23,28 @@ def get(self, lecture_id):
             'comments': comments
         }
 
+    def post(self, lecture_id):
+        db_lectures = Lecture.query.filter(Lecture.id == lecture_id).all()
+
+        if not db_lectures:
+            abort(404, message="Lecture {} does not exist".format(lecture_id))
+
+        parser = reqparse.RequestParser()
+        parser.add_argument('data', help='Text content of comment')
```

Comment:
Yes, I use this rule. Human viewing messages (aka logging, help messages etc) in double quotes, everything else in single. But I don't insist on it, we just have to be consistent :)